### PR TITLE
fix: modify skip button according to feedback comments

### PR DESCRIPTION
**Pull request by tianxiangs**

### CODE REVIEW

File: src/react/components/shell/skipButton.tsx [hunk 1]
```diff
@@ -11,14 +11,18 @@ export class SkipButton extends React.Component<ISkipButtonProps> {
...
             </div>);
     }
 
     private skipToId = (event: React.MouseEvent<HTMLAnchorElement>) => {
         event.preventDefault();
-        const element = document.getElementById(this.props.skipTo);
+
+        const collection = document.getElementsByClassName(this.props.skipTo);
+        const id = collection[0].id;
+        const element = document.getElementById(id);
```
Comment (kunzms) [inline src/react/components/shell/skipButton.tsx:24]: why not element = collection.length > 0 ? collection[0] : null?

Comment (tianxiangs) [reply src/react/components/shell/skipButton.tsx:24]: collection's type is HTMLCollection which cannot call focus function

Comment (tianxiangs) [reply src/react/components/shell/skipButton.tsx:24]: if we can just let element = collection[0], then element.foucus() doesn't exist

File: src/App.tsx [hunk 2]
```diff
@@ -73,6 +73,7 @@ export default class App extends React.Component<IAppProps> {
                     onError={this.props.actions.showError}
                     onClearError={this.props.actions.clearError} />
                 {/* Don't render app contents during a render error */}
+                <SkipButton skipTo="appMainContent">{strings.common.skipToMainContent}</SkipButton>
```
Comment (kunzms) [inline src/App.tsx:76]: suggestion: rename appMainContent to skipToMainContent to make it more clear.

File: src/react/components/shell/skipButton.tsx (hunk 1 above)
Comment (kunzms) [reply src/react/components/shell/skipButton.tsx:27]: convert it to element: collection[0] as HtmlElement

**Pull request merged**